src/piskel-boot.js

```diff
@@ -74,6 +83,6 @@
     }
 
     loadStyle('css/piskel-style-packaged' + version + '.css');
-    loadScript(script, 'onPiskelReady()');
+    loadScript(script, "_onPiskelReady()");
```

Comment:
nit: please use single quotes 
(just realized that files directly under src/ are not linted, will open another issue for that)